rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -637,6 +645,15 @@ def _enrich_protocol_tranfers(  # pylint: disable=no-self-use
             if transfer_enriched:
                 break
 
+    def _enrich_eth_sent(self, tx: EthereumTransaction, event: HistoryBaseEntry) -> None:
```

Comment:
Do we really need a different enricher for ETH sent? Shouldn't we just be using the transfer enricher for all transfers? Including tokens and ETH?

Reply:
Do you mean avoid creating a new type of enricher and just use `_enrich_protocol_tranfers` instead of `_enrich_eth_sent` and have proper checks for the asset in the `_enrich_protocol_tranfers` implementation of the modules that need it?

Reply:
I mean we should have an enricher of all types of transfers. Be it token or ETH transfer. Should not differentiate between the 2.